### DISCUSSION TITLE
How to increase FluxCD Helm Release timeout values

### DISCUSSION THREAD

**Discussion by DevopsCareer** [fluxcd/flux2#3267] [Q&A] [answered]
If Helm Release timesout, what is timeout setting and how can I increase it. My Charts are taking longer to come up and want to understand how can i increase default timeout settings.

**Comment by FileMagic** [comment, marked as answer]
Hey I found the answer, there is a timeout field under the [HelmReleaseSpec](https://v2-0.docs.fluxcd.io/flux/components/helm/api/#helm.toolkit.fluxcd.io/v2beta1.HelmReleaseSpec) object. GLHF.
